contact.html

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Between a Team page, a Registration page, and a Sponsors page, is a contact page necessary?